web/libs/editor/tests/e2e/tests/audio/audio-webaudio-decoder.test.js

```diff
@@ -83,7 +83,7 @@ Scenario('Check if regions are selected', async function({ I, LabelStudio, AtAud
 
   await AtAudioView.waitForAudio();
 
-  I.waitForDetached('loading-progress-bar', 10);
+  I.waitForDetached('loading-progress-bar', 30);
```

Comment:
Do we need to have timeout this high? If so, why has it changed this drastically? Is it somehow related to the change in file storage location?

Reply:
This test failed a couple of times in my local environment due to a timeout, likely because the audio file took longer to load. Since the audio file wasn't loaded after 10 seconds, it resulted in failure. I've adjusted the timeout to ensure that this task doesn't yield a false negative result.